services/registrations/handler.js

```diff
@@ -20,11 +20,9 @@ import {
 */
 export async function updateHelper(data, createNew, email, fname) {
   const {
-    eventID, year, dynamicResponses
+    eventID, year, dynamicResponses, registrationStatus, applicationStatus
   } = data;
   const eventIDAndYear = eventID + ";" + year;
-
-  console.log(data);
```

Comment:
wait small thing here i have a feeling this console log is useful? maybe consider adding it back? @ddennis924